src/app/routes/mod.rs

```diff
@@ -5,6 +5,7 @@ pub mod health;
 pub mod signin;
 pub mod signup;
 pub mod users;
+pub mod webhook;
```

Comment:
Do we need to initialise the webhook in the `init` function in this file?

Reply:
I don't want the webhook route to live behind the "/api" scope that was why I didn't include it here